.github/workflows/release-connect-npm-init.yml

```diff
@@ -36,15 +61,21 @@ jobs:
         run: yarn install
 
       - name: Build dependencies
-        # Running `build:libs` requires action to checkout with `submodules: true`.
         run: yarn build:libs
 
       - name: Run @trezor/connect create npm release branch
         run: |
-          git config --global user.name "trezor-ci"
-          git config --global user.email "${{ secrets.TREZOR_BOT_EMAIL }}"
-          gh config set prompt disabled
-          gh api /user --jq .login
-          node ./ci/scripts/connect-release-init-npm.js ${{ github.event.inputs.semver }}
+          yarn tsx ./ci/scripts/connect-release-npm-init.ts ${{ github.event.inputs.deploymentType }} ${{ env.BRANCH_NAME }}
+        env:
+          BRANCH_NAME: release/connect/${{ needs.extract-version.outputs.version }}
+
+  trigger-npm-release-connect:
```

Comment:
same here with te triggering the action with github actions.

Reply:
I dont think this is resolved

Reply:
You are right, just getting rid of that file completely since this time we are going to run those workflows manually https://github.com/trezor/trezor-suite/pull/12444/commits/95ffe13723f84948396df291ac8c6022c967a6c8